bsd/sys/compat/linux/linux_socket.h

```diff
@@ -151,4 +151,64 @@ struct l_ucred {
 #define	LINUX_IP_ADD_MEMBERSHIP		35
 #define	LINUX_IP_DROP_MEMBERSHIP	36
 
+
+/* Options for use with `getsockopt' and `setsockopt' at the IPv6 level.
+   The first word in the comment at the right is the data type used;
```

Comment:
What does the comment after {get, set}stockpot apply to?  Am I blind?

Reply:
You are right.  Looks like that comment is obsolete.  Not sure where it came from...